PORT-12912 Incorrect Documentation for Dynamic Permissions Logic (#2011)
port-labs/port-docs: docs/actions-and-automations/create-self-service-experiences/set-self-service-actions-rbac/dynamic-permissions.md

  - [`"queries"`](/search-and-query/) - a collection of [rules](/search-and-query/#rules) you can use to fetch and filter the data you need from your software catalog.
  - `"conditions"` - an array of strings, where each string is a `jq` query with access to the `"queries"` data. There is an implicit `"OR"` between each condition.

> If there is **no** `policy`, then `roles`, `users`, and `teams` control who can **view**, **approve**, or **execute** the action.  
> However, If there **is a** `policy`, then `roles`, `users`, and `teams` only control who can **view** the action, while the `policy` exclusively controls who can **execute** and **approve** the action.  

Reviewer: ```suggestion
If there is **no** `policy` object defined, then `roles`, `users`, and `teams` control who can **view**, **approve**, or **execute** the action.  
If the `policy` object **is** defined, then `roles`, `users`, and `teams` only control who can **view** the action, while `policy` exclusively controls who can **execute** and **approve** the action.  
```

> For example, **If no policy is defined**, the following configuration will allow the action to be **both visible and executed** by any user who is either an `Admin` or a member of the `Engineering` team:

Reviewer: ```suggestion
For example, the following configuration (note that no `policy` is defined) will allow the action to be **both visible and executed** by any user who is either an `Admin` or a member of the `Engineering` team:
```

```json
  "execute": {
    "roles": ["Admin"],
    "users": [],
    "teams": ["engineering"]
  }
  ```

> But **If a policy exists**, these roles and teams only determine who can view the action, while the policy exclusively controls who can **execute** or **approve** it.
> 
> For example:

Reviewer: ```suggestion
Using the same configuration, but this time with a `policy` object defined, these `roles` and `teams` only determine who can view the action, while the `policy` exclusively controls who can **execute** or **approve** it.

In the following example, the action will be visible to `Admin` and `Engineering` team members, but its execution permissions depend only on whether the `policy` conditions evaluate to `true`:
```

```json 
"execute": {
  "roles": ["Admin"],
  "users": [],
  "teams": ["engineering"],
  "policy": {
    "queries": {
      "example_query": {
        "rules": [
          // Your rule logic here
        ],
        "combinator": "and"
      }
    },
    "conditions": [
      // A jq query returning a boolean (allowed/not-allowed to execute)
    ]
  }
}
```

> In the above configuration, the action will be visible to `Admin` and `Engineering` team members; however, execution depends only on whether the policy conditions evaluate to true.

Reviewer: ```suggestion
```

> ### Using a policy key

Reviewer: ```suggestion
### Using a policy object
```
